### DISCUSSION TITLE
有没有v3的demo啊

### DISCUSSION THREAD

**Discussion by fhh-coder** [simple-robot/Discussions#65] [Q&A]
我尝试v3, 项目跑不起来，不知道什么原因哦。
项目结构：
![image](https://user-images.githubusercontent.com/57700093/217638608-7ce26642-24e5-4fd4-b313-8ca2860ffc91.png)
![image](https://user-images.githubusercontent.com/57700093/217638655-7d2d01b2-7d6a-48bd-a857-62edaca40d5f.png)
![image](https://user-images.githubusercontent.com/57700093/217638727-391a3ed0-0eaa-4b65-9455-8943f9fd4c1a.png)
![image](https://user-images.githubusercontent.com/57700093/217638790-a37695b8-7e8e-4bec-af96-c68f9775d89b.png)
![image](https://user-images.githubusercontent.com/57700093/217638943-3dd15bfe-d7ea-4437-9f53-8af6c9bcfc5b.png)
日志：
![image](https://user-images.githubusercontent.com/57700093/217639066-936f4fdc-bde7-4ad3-8328-3cf3e2e3f907.png)

**Comment by ForteScarlet** [comment]
https://github.com/simple-robot/simpler-robot/issues/583
